web/src/pages/Route/components/Step1/MatchingRulesView.tsx

```diff
@@ -88,6 +88,18 @@ const MatchingRulesView: React.FC<RouteModule.Step1PassProps> = ({
     });
   };
 
+  const OperatorRenderText = {
+    '==': 'page.route.equal',
```

Comment:
Not a good idea to use `key` here directly, I would prefer keeping the `formatMessage(xxx)` here

Reply:
like `'==': formatMessage({ id: page.route.equal })` ?

Reply:
yes